app/services/deprecation_warnings/disallowed_deprecations.rb

```diff
@@ -11,9 +11,15 @@ class ::DisallowedDeprecationError < StandardError; end
     /Dangerous query method \(method whose arguments are used as raw SQL\) called with non\-attribute argument\(s\)/
   ].freeze
 
+  # Regular expressions for Rails 6.1 deprecation warnings that we have addressed in the codebase
+  RAILS_6_1_FIXED_DEPRECATION_WARNING_REGEXES = [
+    /Controller-level force_ssl is deprecated and will be removed from Rails 6\.1/
```

Comment:
I think we'll need the back-ticks around `force_ssl` in order to match the [deprecation warning](https://github.com/rails/rails/blob/4701a50b586a440d118dce29c9a48bfb306ade5c/actionpack/lib/action_controller/metal/force_ssl.rb#L21C1-L22). Otherwise, this will not work as intended.

```suggestion
    /Controller-level `force_ssl` is deprecated and will be removed from Rails 6\.1/
```